index.html

```diff
@@ -57,7 +57,7 @@
           <a href="https://twitter.com/ElijahHorton" target="_blank" rel="nofollow">@ElijahHorton</a></span>
         </div>
         <div class="heroku-version">
-          <a class="link-to-heroku" href="http://phpepl.herokuapp.com">Click for a PHP 5.5 version</a>
+          <a class="link-to-heroku" href="http://phpepl.herokuapp.com">Click for a PHP 5.6 version</a>
```

Comment:
That sounds good. I'd leave off the `(in Heroku)` version since Heroku is doing away with the free tier and I'd either have to find a free (24/7 uptime with the latest php) alternative or pay for phpepl hosting.

Users don't care about where the app is hosted though, so it's best to leave it off.